src/containers/Login.vue

```diff
@@ -6,20 +6,20 @@
       </div>
     </transition>
 
-    <div class="landing">
-      <!--TOP BAR-->
-      <top-bar />
-
+    <div class="landing top-sec">
       <!--PLAN AHEAD-->
       <div class="container p-0 m-0">
         <div class="row top-bar phonepad no-gutters">
           <div class="col">
             <div class="row no-gutters top-section">
-              <h1 class="plan-head mt-5">Plan Your Courses Ahead</h1>
+              <h1 class="plan-head mt-5">
+                Schedule Smarter <br />
```

Comment:
If you can, can you implement this without the line break element? It is usually considered better practice to use multiple elements and CSS styles to add space between text, especially in case this text or formatting needs to be changed later